runTests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+
+#export SYMFONY_VERSION="~2.7.0"
+SUPPORTED_SYMFONY_VERSIONS=('~2.3.0' '~2.5.0' '~2.6.0' '~2.7.0' '2.8.x-dev')
+export DOCTRINE_VERSION=">=2.2.3,<2.5-dev"
+
+sudo apt-get -qq update
```

Comment:
This shouldn't be done here.

I like the general idea of this script but it should only be responsible for launching the tests (that includes playing with the `composer.json` file and optionally downloading composer).